infrastructure/logging/src/main/java/tech/pegasys/teku/infrastructure/logging/EventLogger.java

```diff
@@ -194,6 +194,26 @@ public void terminalPowBlockDetected(final Bytes32 terminalBlockHash) {
         Color.GREEN);
   }
 
+  public void transitionConfiguration_TTD_TBH_mismatch(
```

Comment:
We shouldn't be using snake case in method names. In standard java form this would be:
```suggestion
  public void transitionConfigurationTtdTbhmismatch(
```